bravado/mapping/param.py

```diff
@@ -128,6 +132,73 @@ def marshal_param(param, value, request):
             format(location))
```

Comment:
A question on new version's L128 above, for `json.dumps` it would expect a `dict` as a param. So, will this be no longer valid now: `client.pet.addPet(body=Pet(id=1, name='bla')).result()` ?